### DISCUSSION TITLE
클라우드컴퓨팅 구현 이펙트 관련 토의

### DISCUSSION THREAD

**Discussion by YOON-JAEYEONG** [YOON-JAEYEONG/Student_Survivor#27] [Ideas]
@hasunghun @rlatldnd1 @AolnD @asdf-1256 @gnodss 

여유되실 때 이펙트 관련해서 스킬 회의할 때 생각하고 있었던 의견같은거 있으면 남겨주시고 그 중에 하나 골라서 만들죠.

구현 난이도같은건 일단 고려하지말고

추가)자세하게 써주시면 더 좋아용 머릿속으로 그려지게!

**Comment by rlatldnd1** [comment]
알림 바로 오네

**Reply by YOON-JAEYEONG** [reply]
멘션해서 그래용

**Comment by YOON-JAEYEONG** [comment]
![Notes_231006_224519.jpg](https://github.com/asdf-1256/Student_Survivor/assets/115152348/5b0eafc6-e7b5-4612-bf2e-01371466e2a5)

제가 생각한건 이런 느낌이었습니다

**Reply by AolnD** [reply]
구름이 터지면서 구름조각들이 날라가는??
구름이 점점 커지면서 광역폭우(노트북떨어짐)

**Reply by YOON-JAEYEONG** [reply]
글씨체가 더러워서 줄글로 풀어서 정리해봤어요


쿨타임이 지나서 스킬이 발동되면 플레이어에게서 구름이 소환된다. 소환된 구름은 적의 머리 위로 이동 후 움직임을 멈추고, 전자기기를 하단 60도 정도 각도 내에서 랜덤으로 발사한다. 발사된 전자기기는 적과 피격되거나, 일정 범위에서 벗어나면 소멸한다. 전자기기 발사를 완료하고 전자기기가 모두 사라지면 구름이 서서히 사라진다 (투명도가 올라가는 느낌으로). 구름은 적과 피격 시에 데미지를 주지 않으며, 전자기기 만이 데미지를 줄 수 있다.

**Reply by gnodss** [reply]
쿨타임 지나서 스킬 발동될때마다 하나씩 구름이 맵에서 랜덤하게 생성되서 그 부분에서 몇초동안 노트북이나 뭐든 떨어지고 나서 사라지고 하는 식으로
업그레이드 되면 구름 개수 늘려도되고 크기 늘려도되구

**Reply by rlatldnd1** [reply]
☁️ 적들 머리 위에서 빠르게 이동하면서 주기적으로 전자기기를 떨어뜨린다. 공격할 적은 항상 랜덤하게 타겟으로 선택
타겟이 된 적에게 물리적으로 다가가며, 해당 타겟이 죽었을 경우 즉시 새로운 타겟을 탐지한다.

**Comment by hasunghun** [comment]
하이용
